armsrc/spiffs.c

```diff
@@ -533,21 +539,16 @@ void rdv40_spiffs_safe_print_fsinfo() {
     rdv40_spiffs_fsinfo fsinfo;
     rdv40_spiffs_getfsinfo(&fsinfo, RDV40_SPIFFS_SAFETY_SAFE);
     DbpString(_BLUE_("Flash Memory FileSystem Info (SPIFFS)"));
-//    Dbprintf("-------------------------------------");
-    Dbprintf("  Logical Block Size........." _YELLOW_("%d")"bytes", fsinfo.blockSize);
-    Dbprintf("  Logical Page Size.........." _YELLOW_("%d")"bytes", fsinfo.pageSize);
-    Dbprintf("");
-    Dbprintf("  Max Open Files............." _YELLOW_("%d")"file descriptors", fsinfo.maxOpenFiles);
-    Dbprintf("  Max Path Length............" _YELLOW_("%d")"chars", fsinfo.maxPathLength);
-//    DbpString(_BLUE_("Details"));
-    DbpString("");
-    Dbprintf("  Filesystem\tSize\tUsed\tAvailable\tUse%\tMounted on");
-    Dbprintf("  spiffs    \t%d B\t%d B\t%d B\t\t"_YELLOW_("%d%")"\t/"
-             , fsinfo.totalBytes
-             , fsinfo.usedBytes
-             , fsinfo.freeBytes
-             , fsinfo.usedPercent
-            );
```

Comment:
the "\t" tabs works very bad on WSL.   I strongly suggest consistent use of 4 spaces instead of tab.

Reply:
Are you seriously using WSL as an argument in about how should TTy-formating be done ?
:D 
More seriously, Feel free to revert back some change that my cherrypicks may have lost and to show how consistant space allways had been through space and time.
woooossh

Reply:
.... yes....   because I am naughty.

There is some string format woodoo to get fixed cols instead of using tabs.

Reply:
FYI there is a ton of files with such printf \t
./client/crypto/asn1dump.c
./client/proxguiqt.cpp
./client/mifare/ndef.c
./client/mifare/mifarehost.c
./client/reveng/cli.c
./client/reveng/poly.c
./client/reveng/bmpbit.c
./client/jansson/dump.c
./client/jansson/load.c
./client/jansson/pack_unpack.c
./client/loclass/hash1_brute.c
./client/loclass/cipherutils.c
./client/emv/emv_tags.c
./client/emv/test/cryptotest.c
./client/emv/cmdemv.c
./client/emv/dump.c
./client/cmdlf.c
./client/cliparser/cliparser.c
./client/cliparser/argtable3.c
./client/cmdhfmfp.c
./client/cmdhficlass.c
./client/tinycbor/cborpretty.c
./client/scripts/Legic_clone.lua
./client/scripts/ntag_3d.lua
./client/scripts/legic.lua
./client/cmdhfmfu.c
./client/lualibs/default_toys_di.lua
./client/lualibs/dkjson.lua
./client/lualibs/default_toys.lua
./client/hardnested/hardnested_bruteforce.c
./client/cmdhffelica.c
./client/pm3_mfdread.py
./client/flasher.c
./client/cmdlfindala.c
./client/cmdhf15.c
./client/cmdsmartcard.c
./client/cmdhffido.c
./client/cmdhfmf.c
./client/liblua/lauxlib.c
./client/liblua/lbaselib.c
./client/liblua/llex.c
./client/liblua/luac.c
./client/liblua/loadlib.c
./client/cmdhf14b.c
./client/cmdhf14a.c
./client/util.c
./armsrc/spiffs_hydrogen.c
./armsrc/spiffs.c
./armsrc/frozen.c
./armsrc/Standalone/hf_colin.c
./armsrc/Standalone/hf_mattyrun.c
./fpga/tests/tb_lf_edge_detect.v
./fpga/tests/tb_min_max_tracker.v
./tools/deprecated-hid-flasher/flasher/flasher.c
./tools/deprecated-hid-flasher/flasher/proxusb.c
./tools/analyzesize.py
./tools/xorcheck.py
./tools/findbits.py
./common/mbedtls/bn_mul.h